pkg/storage/tsdb/expanded_postings_cache.go

```diff
@@ -53,6 +54,10 @@ func NewPostingCacheMetrics(r prometheus.Registerer) *ExpandedPostingsCacheMetri
 			Name: "cortex_ingester_expanded_postings_cache_hits",
 			Help: "Total number of hit requests to the cache.",
 		}, []string{"cache"}),
+		CacheMiss: promauto.With(r).NewCounterVec(prometheus.CounterOpts{
+			Name: "cortex_ingester_expanded_postings_cache_miss",
```

Comment:
Just realized we need to add `_total` to all counter metrics. Can you change all metrics here?

`cortex_ingester_expanded_postings_cache_missed_total`

Reply:
done